hudi-utilities/src/test/java/org/apache/hudi/utilities/functional/TestHoodieDeltaStreamer.java

```diff
@@ -250,7 +250,7 @@ static HoodieDeltaStreamer.Config makeConfig(String basePath, WriteOperationType
       cfg.operation = op;
       cfg.enableHiveSync = enableHiveSync;
       cfg.sourceOrderingField = sourceOrderingField;
-      cfg.propsFilePath = dfsBasePath + "/" + propsFilename;
+      cfg.propsFilePath = UtilitiesTestBase.basePath + "/" + propsFilename;
```

Comment:
basePath is protected in UtilitiesTestBase and TestHoodieDeltastreamer extends from it. so why prefixing with class name ?

Reply:
because there is another local var named the same. don't want to rename more variables further.